provider/resources.go

```diff
@@ -5718,6 +5719,24 @@ compatibility shim in favor of the new "name" field.`)
 			args.ExamplePath == "#/resources/aws:appsync/graphQLApi:GraphQLApi"
 	}
 
+	// TODO[pulumi/pulumi-terraform-bridge#2938]
+	// Currently, the bridge omits write-only attributes from being written to a provider schema.
+	// In this provider, we additionally have some attributes  called things like "has_value_wo" or "version_wo".
+	// To avoid user confusion, we omit these here as well.
+	tfbridge.MustTraverseProperties(&prov, "write-only-supporting-properties",
+		func(propertyVisitInfo tfbridge.PropertyVisitInfo) (tfbridge.PropertyVisitResult, error) {
+			schemaPath := propertyVisitInfo.SchemaPath()
+			if key, ok := schemaPath[len(schemaPath)-1].(walk.GetAttrStep); ok {
+				if strings.Contains(key.Name, "_wo_") || strings.HasSuffix(key.Name, "_wo") {
```

Comment:
I like this, this seems reasonable. If we are worried that this will over-eagerly skip a property that does not need skipping,  we can add a file with a checklist of removed properties, say (resource, SchemaPath) pairs and force PRs to change that, perhaps in a test.